sdk/core/azure-core/examples/test_example_async.py

```diff
@@ -33,6 +34,12 @@
 
 import trio
 
+if sys.version_info < (3, 5):
+    RUN_TESTS = True
+else:
+    RUN_TESTS = False
+
+@pytest.mark.skipif(RUN_TESTS, reason="python version < 3.5")
```

Comment:
Just noting this convention surprised me -- I'd expect that if `RUN_TESTS` is True, then the tests will be run, not skipped.

Reply:
Ah - good point. I actually forgot to delete this check since it doesn't work with the CI and the check for python < 3.5 is now done in the conftest.py file.

I'll remove it.